app/experimenter/nimbus-ui/src/components/PageResults/TableMetricSecondary/index.tsx

```diff
@@ -56,11 +60,39 @@ const TableMetricSecondary = ({
     : METRIC_TYPE.USER_SELECTED_SECONDARY;
 
   const overallResults = results?.overall!;
+  const probeSetName =
+    results.metadata?.metrics[probeSetSlug]?.friendly_name ||
+    probeSetDefaultName;
+  const probeSetDescription =
+    results.metadata?.metrics[probeSetSlug]?.description || undefined;
 
   return (
     <div data-testid="table-metric-secondary" className="mb-5">
       <h2 className="h5 mb-3" id={probeSetSlug}>
-        <div>{probeSetName}</div>
+        <div>
+          <div className="d-inline-block">
+            {probeSetName}{" "}
+            {probeSetDescription && (
+              <>
+                <Info
+                  data-tip
+                  data-for={probeSetSlug}
+                  className="align-baseline"
+                />
+                <ReactTooltip
+                  id={probeSetSlug}
+                  multiline={true}
+                  clickable={true}
```

Comment:
This doesn't appear to actually be clickable from what I can [see](https://storage.googleapis.com/mozilla-storybooks-experimenter/commits/78ee0b46aac30477c43555b21a73a47ac5d07ad4/nimbus-ui/index.html?path=/story/pages-results--basic)? That's fine with me, I prefer it to be consistent with the other tooltips, but it looks like we're trying to make this clickable and it isn't.

Reply:
Also props that are just `true` should be able to simply be just the prop, so `multiline clickable`

Reply:
I've updated it to remove `{true}` and I added a link so it's more clear that it's clickable.